src/shared/forminator/inputs/SelectMultiAutoComplete.tsx

```diff
@@ -38,15 +38,20 @@ function SelectMultiAutoComplete<Suggestion extends BaseSuggestion = BaseSuggest
   label,
   textFieldOptions,
   excludes,
-  maximumValues = Number.POSITIVE_INFINITY,
-  messageMaximumReviewers = '',
+  maximumValues: maximumValuesProp = -1,
```

Comment:
why you not use `Number.POSITIVE_INFINITY` directly here

Reply:
Because if there is no limit. the back-end will send -1